tests/Query/OrderbyTestCase.php

```diff
@@ -35,13 +35,14 @@ class Doctrine_Query_Orderby_TestCase extends Doctrine_UnitTestCase
     public function testOrderByRandomIsSupported()
```

Comment:
non-whitespace changes in this file are just fixing some type-os (I was going to add a test to this file for #37, but ended up not needing to).